kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/utils/SqlUtils.java

```diff
@@ -0,0 +1,294 @@
+package fr.insee.kraftwerk.core.utils;
+
+import fr.insee.kraftwerk.core.Constants;
+import fr.insee.kraftwerk.core.metadata.VariableType;
+import fr.insee.kraftwerk.core.vtl.VtlBindings;
+import fr.insee.vtl.model.Dataset;
+import fr.insee.vtl.model.Structured;
+import lombok.extern.slf4j.Slf4j;
+import org.duckdb.DuckDBConnection;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+@Slf4j
+public class SqlUtils {
+
+    private SqlUtils() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    /**
+     * Convert vtl bindings to SQL DuckDB tables for further export
+     * @param vtlBindings vtl bindings to send into database
+     * @param statement statement associated to database
+     */
+    public static void convertVtlBindingsIntoSqlDatabase(VtlBindings vtlBindings, Statement statement) {
+        try {
+            createOutputSQLTables(statement, vtlBindings);
+        } catch (SQLException e) {
+            log.error("SQL Error during VTL bindings conversion :\n{}",e.toString());
+        }
+    }
+
+
+    /**
+     * send CREATE TABLE queries into DB
+     *
+     * @param statement   DuckDB connection
+     * @param vtlBindings datasets to convert
+     * @throws SQLException if sql error
+     */
+    private static void createOutputSQLTables(Statement statement, VtlBindings vtlBindings) throws SQLException {
```

Comment:
change method name => output is used for final output (csv, parquet)... createSchemaForSqlTables ?